packages/istanbul-reports/lib/html/annotator.js

```diff
@@ -41,9 +41,11 @@ function annotateLines(fileCoverage, structuredText) {
 }
 
 function annotateStatements(fileCoverage, structuredText) {
-    var statementStats = fileCoverage.s,
-        statementMeta = fileCoverage.statementMap;
-    Object.keys(statementStats).forEach(function (stName) {
+    var statementStats = Object.keys(fileCoverage.s).filter(function (stName) {
+      return fileCoverage.s[stName] !== null;
```

Comment:
would it be overly poor form to use `!= null` to capture any ones that are `undefined` as well? or would it be overkill to do `&& !== undefined`?